lib/util/ts_parser.js

```diff
@@ -996,21 +1001,37 @@ shaka.util.TsParser = class {
     videoInfo.codec = 'avc1.' + this.byteToHex_(profileIdc) +
         this.byteToHex_(profileCompatibility) + this.byteToHex_(levelIdc);
 
+    const videoData = this.getVideoData();
```

Comment:
This exact same code block is also now in `getHvcInfo_`.
It could be broken into a utility function, that is passed the videoInfo and applies the frameRate to it if one is found by `getVideoData`.
`getVideoDataAndApplyFramerate_(videoInfo)`, maybe?